scripts/ci/lib.sh

```diff
@@ -631,7 +631,7 @@ mark_collector_release() {
 
     # We need to make sure the file ends with a newline so as not to corrupt it when appending.
     [[ ! -f RELEASED_VERSIONS ]] || sed --in-place -e '$a'\\ RELEASED_VERSIONS
-    if grep -q "${tag}" RELEASED_VERSIONS; then
+    if grep -q "\s${tag}\s\s" RELEASED_VERSIONS; then
```

Comment:
I cannot find any grep-docs about `\s`. This must be some bash escaping, right?

I would suggest to use the built-in grep option instead:

```suggestion
    if grep -qF "${tag}" RELEASED_VERSIONS; then
```
 
it does: 

```
       -F, --fixed-strings
              Interpret PATTERNS as fixed strings, not regular
              expressions.
```

Reply:
That works as well.